stdlib/runtime/string.gr

```diff
@@ -367,7 +367,7 @@ let rec heapValueToString = (ptr, extraIndents, toplevel) => {
         needsEllipsis = true
       }
       let headBytes = 8n // <bytes:
-      let hexBytes = numBytes * 3n
+      let hexBytes = numBytes * 3n - 1n
```

Comment:
Can you add a comment to this explaining that this is two digits + a space for each byte, minus one for the extra space at the end?